sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/TextAnalyticsClientTest.recognizeLinkedEntitiesForTextInput[1].json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Do we need to record all the recordings for `v3.1-preview.3` endpoint?
Why only these?

Reply:
I only record these tests because these test failed. We don't need to record all. Old RECORD JSON files works.